scripts/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${RUNNER_NAME}" ]; then
+  echo "RUNNER_NAME must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_TOKEN}" ]; then
+  echo "RUNNER_TOKEN must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_ORG}" ]; then
+  echo "RUNNER_ORG must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_REPO}" ]; then
+  echo "RUNNER_REPO must be set" 1>&2
+  exit 1
+fi
+
+cd /runner
+mkdir -p _work
+./config.sh --unattended --replace --name "${RUNNER_NAME}" --url "https://github.com/${RUNNER_ORG}/${RUNNER_REPO}" --token "${RUNNER_TOKEN}" --work /runner/_work
+
+# TODO: run placemat
```

Comment:
Just to be clear, we're going to run Placemat and neco bootstrap inside this runner Pod and kick start a user-defined workflow when it's assigned. Am I correct?

Reply:
Yes, and if the initializing step fails, this script executes `kubectl annotate ...` and heve the sweeper delete the Pod.
For now, I don't makethis initializing logic pluggable or user-defined for simplicity. If we wanna do so, this should be achived with `ConfigMap` or `initContainers` or something.